### ISSUE TITLE
Automatic mode does not work in v10

### ISSUE THREAD

**Issue by cs96and** [sPOiDar/fvtt-module-stream-view#39]
I just updated to Foundry v10, and updated the Stream View module to v1.3.0.  Since updating, I cannot get automatic mode to work.  No matter which mode I choose, it always works like it is in "directed" mode.  I have tried disabling all modules except for Stream View, lib-wrapper and socketlib, and also tried downgrading Stream View to v1.0.0 but still have the same problem.

Also, Stream View only seems to follow canvas movements on the activated scene.  E.g.

1. Activate scene 1
2. As GM move to Scene 2 (but don't activate it)
3. Pull stream user to scene 2
4. Move the camera around.  No changes are made to the stream user's view
5. As GM, move back to scene 1
6. Move the camera around.  The camera moves on the stream user's view

**Comment by sPOiDar** [comment]
Could you please split the second feature out to a new issue so I can track them separately? :heart: 

As for the first issue, I can't reproduce it here, though I don't think I have an instance handy that was upgraded from v9 to v10 in-place.

Please check the javascript console for any errors on both the GM and stream views by hitting F12 in your browser. Please try toggling between camera modes using the button in the Stream View toolbar while doing this.

Also, please grab the current settings from the module so that I can test with exactly the same values locally in case there's some combo that can trigger this, by executing the following in the console:

```javascript
game.settings.settings.forEach((value, key) => key.startsWith('stream-view') && console.log(key, game.settings.get(value.namespace, value.key)))
```

**Comment by cs96and** [comment]
I discovered that there was an unlinked combat hanging around in my game (no idea where it cam from, all the combatants were "Unknown"), and I had Directed Combat enabled.  After deleting the encounter, it started working properly again, when out of combat.

I also had to disable "Directed Combat" to make it work properly in combat.  I don't understand that option.  The description makes it sound like you must enable that option to move between combatants when the turn changes, but it seems to have the opposite behaviour to me.
